Python/Tests/Utilities.Python.Analysis/PythonPaths.cs

```diff
@@ -120,14 +127,32 @@ public class PythonPaths {
                 if (Python35 != null) yield return Python35;
                 if (Python36 != null) yield return Python36;
                 if (Python37 != null) yield return Python37;
-                if (Python38 != null) yield return Python37;
-                if (Python39 != null) yield return Python37;
+                if (Python38 != null) yield return Python38;
+                if (Python39 != null) yield return Python39;
                 if (Python27_x64 != null) yield return Python27_x64;
                 if (Python35_x64 != null) yield return Python35_x64;
                 if (Python36_x64 != null) yield return Python36_x64;
                 if (Python37_x64 != null) yield return Python37_x64;
-                if (Python38_x64 != null) yield return Python36_x64;
-                if (Python39_x64 != null) yield return Python37_x64;
+                if (Python38_x64 != null) yield return Python38_x64;
+                if (Python39_x64 != null) yield return Python39_x64;
+            }
+        }
+
+        public static PythonVersion LatestVersion {
```

Comment:
This property makes the tests runnable with whatever the latest Python is on your box.